tests/kubectl_test.go

```diff
@@ -50,9 +56,15 @@ var _ = Describe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kub
 	table.DescribeTable("should verify set of wide columns for", func(verb, resource, option string, expectedHeader []string, verifyPos int, expectedData string) {
 
 		result, _, _ := tests.RunCommand(k8sClient, verb, resource, "-o", option)
-
-		Expect(result).ToNot(BeNil())
+		// due to issue of kubectl that sometimes doesn't show CRDs on the first try, retry the same command
+		if err != nil {
+			result, _, err = tests.RunCommand(k8sClient, verb, resource)
```

Comment:
You are missing the `-o, option` args option here.
Also, wouldn't an Eventually block make sense, that would prevent such mistakes?

Reply:
@slintes good catch for the `-o`, corrected it now
About `Eventually`, the error should ideally occur only first time on a fresh cluster, so only one retry here makes sense, after that more tries would be some bug.

Reply:
"ideally" sounds like "eventually", doesn't it? ;) You can retry once with it, and it removes code duplication. But I won't block on this.